parse.go

```diff
@@ -55,9 +55,17 @@ var (
 	ErrorMismatchPixelDataLength = errors.New("the size calculated from DICOM elements and the PixelData element's VL are mismatched")
 )
 
+func Parse(in io.Reader, bytesToRead int64, frameChan chan *frame.Frame, opts ...ParseOption) (Dataset, error) {
+	return parseInternal(in, bytesToRead, frameChan, opts...)
+}
+
+func ParseUntilEOF(in io.Reader, frameChan chan *frame.Frame, opts ...ParseOption) (Dataset, error) {
```

Comment:
Can you add tests for this similar to https://github.com/suyashkumar/dicom/blob/f77ffdec9a83ffddf27d05aeaeb97f7fb220e589/parse_test.go#L22

This code path also isn't covered by the roundtrip tests [here](https://github.com/suyashkumar/dicom/blob/f77ffdec9a83ffddf27d05aeaeb97f7fb220e589/write_test.go#L30) that would also be good to have. 

One option would be to modify the body of the test so that every test case is run twice--one in the normal Parse mode, and another in the ParseUntilEOF mode to exercise it.

Reply:
I'm adding TestParseUntilEOF that will do just that

Reply:
Thanks! Left a comment on the test about a possible addition I think would be helpful.